src/applications/personalization/appointments/actions/index.js

```diff
@@ -142,12 +57,14 @@ export function fetchConfirmedFutureAppointmentsV2() {
       const formatted = appointments.map(appointment => {
         return vaosV2Helpers.transformAppointment(appointment);
       });
-      // sort by date
-      const sorted = vaosV2Helpers.sortAppointments(formatted);
```

Comment:
I see that you are filtering out the past appointments, but is there still a need to sort the appointments?

Reply:
oh man i read this wrong 🤦 i didn't notice that these lines were deleted

yes there is indeed still a need to sort appointments. going to open a PR back up to add this back in.